### DISCUSSION TITLE
Error KICS Auto Scanning with VS Code

### DISCUSSION THREAD

**Discussion by fernandomxm** [Checkmarx/kics#6620] [Q&A] [answered]
When i try execute Auto Scanning with VS Code, presents this error:

Error: json: cannot unmarshal number -1 into Go struct field KicsFiles.queries.files.search_line of type uint

Has anyone had this problem?

**Comment by cx-pedro-lopes** [comment, marked as answer]
Hi @fernandomxm  We just fixed this issue in the new version of vscode. Please update to [v2.3.0](https://github.com/Checkmarx/ast-vscode-extension/releases/tag/v2.3.0)
